src/analysis/normed_space/banach.lean

```diff
@@ -400,3 +400,76 @@ begin
 end
 
 end continuous_linear_map
+
+section closed_graph_thm
+
+variables [complete_space E] (g : E →ₗ[𝕜] F)
+
+/-- The **closed graph theorem** : a linear map between to Banach spaces whose graph is closed
+is continuous. -/
+theorem linear_map.continuous_of_is_closed_graph (hg : is_closed (g.graph : set $ E × F)) :
+  continuous g :=
+begin
+  letI : complete_space g.graph := complete_space_coe_iff_is_complete.mpr hg.is_complete,
+  let φ₀ : E →ₗ[𝕜] E × F := linear_map.id.prod g,
+  have : function.left_inverse prod.fst φ₀ := λ x, rfl,
+  let φ : E ≃ₗ[𝕜] g.graph :=
+    (linear_equiv.of_left_inverse this).trans
+    (linear_equiv.submodule_congr g.graph_eq_range_prod.symm),
+  let ψ : g.graph ≃L[𝕜] E := φ.symm.to_continuous_linear_equiv_of_continuous
+    continuous_subtype_coe.fst,
+  exact (continuous_subtype_coe.comp ψ.symm.continuous).snd
+end
+
+/-- A useful form of the **closed graph theorem** : let `f` be a linear map between two Banach
+spaces. To show that `f` is continuous, it suffices to show that for any convergent sequence
+`uₙ ⟶ x`, if `f(uₙ) ⟶ y` then `y = f(x)`. -/
+theorem linear_map.continuous_of_seq_graph
```

Comment:
If anyone has a better name idea for that, please tell me ! Of course they'd be `continuous_of_eq_of_tendsto` but I'd like something clearer

Reply:
I don't think `seq_graph` is understandable. `seq_closed_graph`?